lib/browser/app.js

```diff
@@ -59,6 +59,12 @@ app.controller('AppController', function($q, DriveScannerService, SelectionState
     this.selection.clear();
     this.writer.setProgress(0);
     this.scanner.start(2000);
+
+    this.scanner.scan().then(function (res) {
```

Comment:
Really low priority, but we don't put a space between the `function` keyword and the open parenthesis in JavaScript.